README.md

```diff
@@ -59,7 +59,8 @@ custom:
         DurationPc:
```

Comment:
Can we update the config in the README to show the default values for everything (in default-config) including enabled: true?
This way, it's clear to see what the default config is and it documents the supported values at the same time.
It might be worthwhile at some point using YAML for `default-config` and building it into the README. what do you think?

Reply:
I can easily update the readme, but certainly good idea on the yaml!